draccus/wrappers/docstring.py

```diff
@@ -18,6 +18,28 @@ class AttributeDocString:
     docstring_below: str = ""
 
 
+# Default Help Orderings
+HELP_ORDERS = {
+    "inline": ["comment_inline", "comment_above", "docstring_below"],
```

Comment:
imho docstring_below should be the first backup for this and "above": if you're putting a string right after a field, it is probably intended as the doc for the field

Reply:
10-4, resolved.